Monika After Story/game/script-songs.rpy

```diff
@@ -0,0 +1,302 @@
+default persistent._mas_songs_database = dict()
+
+init -10 python in mas_songs:
+    # Event database for songs
+    song_db = {}
+
+    #Song type constants
+    TYPE_LONG = "long"
```

Comment:
can you add comments on what a long song is like vs a short one? this would help remind us the criteria we originally had in mind when deciding these constants.